app/Http/Controllers/CampaignController.php

```diff
@@ -222,6 +222,16 @@ public function leave()
         }
     }
 
+    /**
```

Comment:
Seems like this should be removed? :)

Reply:
If I remove it, the notifications stop working, the job gets processed but the notifications are never sent to the campaign admins.